app/src/main/java/com/google/droidjump/GameSuccessFragment.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.droidjump;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageButton;
+
+import androidx.navigation.Navigation;
+
+import java.util.Objects;
+
+
+public class GameSuccessFragment extends Fragment {
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View rootView = inflater.inflate(R.layout.game_success_screen, container, false);
+        FloatingActionButton nextLevelButton = rootView.findViewById(R.id.next_button);
+        nextLevelButton.setOnClickListener(view -> {
+            Navigation.findNavController(view).navigate(R.id.action_game_success_screen_to_game_screen);
```

Comment:
can we use findNavController(view).navigate(R.id.action_game_success_screen_to_game_screen); instead of Navigation.findNavController(view).navigate(R.id.action_game_success_screen_to_game_screen);?

Reply:
if yes, would you mind changing everywhere?

Reply:
We can but we would need to use static import **androidx.navigation.Navigation.findNavController** . As I know, using static imports is not a good practice. @redls what should I do?

Reply:
Usage of static imports is a considered a good practice in Google if the usage is clear without additional context of the enclosing type. Static imports serve to remove redundant context and reduce boilerplate. The method/field should be fairly well-known, and self-describing.

I think in this case "findNavController" speaks for it itself. I would personally prefer a static import here, but this is a nitpick for sure.

Reply:
Thanks!